runtime/compiler/x/codegen/X86PrivateLinkage.cpp

```diff
@@ -1667,7 +1667,7 @@ TR::Register *TR::X86PrivateLinkage::buildIndirectDispatch(TR::Node *callNode)
          */
          TR::MemoryReference  *sourceMR = generateX86MemoryReference(vftChild, cg());
          TR::Register *reg = cg()->allocateRegister();
-         // as vftChild->getOpCode().isLoadIndirect is true here, need set excpetionpoint
+         // as vftChild->getOpCode().isLoadIndirect is true here, need set exceptionpoint
```

Comment:
Should this be exceptionPoint or ExceptionPoint? @fjeremic

Reply:
Should be "exception point".